content/graphing/dashboards/screenboard.md

```diff
@@ -83,11 +85,12 @@ Here is how to proceed to enjoy this css style on Chrome using Stylebot:
 
 If you have any custom css style sheet that you would like to share, [reach out to us](/help)!
 
-## Backup my Screenboard
+## Back up my Screenboard
```

Comment:
No, i think the proper term is backup. back up doesn't make sense in this context.